tests/acceptance/features/apiSharingNgShareInvitation/updateShareInvitations.feature

```diff
@@ -717,7 +717,7 @@ Feature: Update permission of a share
       | sharee             | grp1                 |
       | shareType          | group                |
       | permissionsRole    | <permissions-role>   |
-      | expirationDateTime | 2025-07-15T14:00:00Z |
+      | expirationDateTime | tomorrow             |
     When user "Alice" updates the last drive share with the following using root endpoint of the Graph API:
       | expirationDateTime | 2200-07-15T14:00:00Z   |
```

Comment:
Maybe you want to adjust that as well ?

Reply:
Maybe just simplify the process and adjust the tests to a different date. e.G 2200- which was done already in other tests.

Reply:
This specific test checks the date in the JSON response, so if you want to make it dynamic there one would have to teach the step `Then the JSON data of the response should match` to interpret `tomorrow`. e.g. by creating a new substitution.